xlators/mgmt/glusterd/src/glusterd-volume-set.c

```diff
@@ -2932,6 +2930,17 @@ struct volopt_map_entry glusterd_volopt_map[] = {
                     "brick multiplexing. Brick multiplexing ensures that "
                     "compatible brick instances can share one single "
                     "brick process."},
+    /*{.key = GLUSTER_BRICK_GRACEFUL_CLEANUP,
+     .voltype = "mgmt/glusterd",
+     .value = "disable",
+     .op_version = GD_OP_VERSION_9_0,
+     .validate_fn = validate_boolean,
+     .type = GLOBAL_DOC,
+     .description = "This global option can be used to enable/disable "
+                    "graceful cleanup, after enable graceful cleanup "
+                    "glusterd always sends a detach rpc request to stop a "
+                    "brick process."},
+    */
```

Comment:
Why it's commented ? we need the option, only don't make it public with the `GLOBAL_DOC` flag...

Reply:
After doing a regress testing I will send a new patch to make it public then i will update it.

Reply:
done